source/backend/entities/ef/PimsAccessRequestHist.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
could you remove this changes from the PR? These files are auto-generated and will be overridden when the next scaffold is run

Reply:
good call. I've regenerated the entities to clear the changes